packages/application-extension/src/index.tsx

```diff
@@ -142,6 +142,12 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
     // Add Command to override the JLab context menu.
     commands.addCommand(JupyterFrontEndContextMenu.contextMenu, {
       label: trans.__('Shift+Right Click for Browser Menu'),
+      describedBy: {
+        args: {
+          type: 'object',
+          properties: {}
+        }
+      },
```

Comment:
```suggestion
      describedBy: {},
```

It seems that Lumino allows [empty object](https://github.com/jupyterlab/lumino/blob/20cf366a88028ca763eb80241bd1226c936f454e/packages/commands/src/index.ts#L833).
Perhaps this would reduce the source a little, since many commands have no arguments, by keeping the `describedBy` property to differentiate them from undescribed commands.

Reply:
Thanks for the suggestion 👍 

Yeah that sounds like a fair tradeoff, happy to push a commit implementing and see what people think.